apps/fyler/src/fyler_event_listener.erl

```diff
@@ -26,26 +26,11 @@ init(_Args) ->
   ?D({event_handler_set}),
   {ok,[]}.
 
-handle_event(#fevent{type = complete, node = Node, task = #task{id=TaskId, file = #file{url = Url}, type = Type, category = Category} = Task, stats = #job_stats{time_spent = Time, download_time = DTime} = Stats}, State) ->
-  ?D({task_complete, Type, Url, {time,Time},{download_time,DTime}}),
-  case ets:info(Category) of
-    undefined ->
-      {ok, State};
-    _ ->
-      case ets:member(Category, TaskId) of
-        true ->
-          ets:delete(Category, TaskId),
-          fyler_server:save_task_stats(Stats),
-          fyler_server:send_response(Task,Stats,success),
-          gen_server:cast(fyler_server, {task_finished,Node}),
-          {ok, State};
-        false ->
-          {ok, State}
-      end
-  end;
+handle_event(#fevent{type = Result, node = Node,
+  task = #task{id = TaskId, file = #file{url = Url, size = Size}, type = Type, category = Category} = Task,
```

Comment:
Line 30 is too long:   task = #task{id = TaskId, file = #file{url = Url, size = Size}, type = Type, category = Category} = Task,.